contracts/RiskManagerV1.sol

```diff
@@ -472,3 +752,191 @@ contract RiskManagerV1 is IRiskManager, Auctioneer, Ownable {
+    }
+
+    /// @notice Finalizes the liquidation notifier reward percentage update process.
+    function finalizeLiquidationNotifierRewardPercentageUpdate(
+        Storage storage self
+    ) internal {
+        self.liquidationNotifierRewardPercentage = self
+            .newLiquidationNotifierRewardPercentage;
+        self.newLiquidationNotifierRewardPercentage = 0;
+        self.liquidationNotifierRewardPercentageChangeInitiated = 0;
+    }
+
+    /// @notice Begins the liquidated notifier reward amount update process.
+    /// @param _newLiquidatedNotifierRewardAmount New liquidated notifier
+    ///        reward amount.
+    function beginLiquidatedNotifierRewardAmountUpdate(
+        Storage storage self,
+        uint256 _newLiquidatedNotifierRewardAmount
+    ) internal {
+        /* solhint-disable not-rely-on-time */
+        self
+            .newLiquidatedNotifierRewardAmount = _newLiquidatedNotifierRewardAmount;
+        self.liquidatedNotifierRewardAmountChangeInitiated = block.timestamp;
+        /* solhint-enable not-rely-on-time */
+    }
+
+    /// @notice Finalizes the liquidated notifier reward amount update process.
+    function finalizeLiquidatedNotifierRewardAmountUpdate(Storage storage self)
+        internal
+    {
+        self.liquidatedNotifierRewardAmount = self
+            .newLiquidatedNotifierRewardAmount;
+        self.newLiquidatedNotifierRewardAmount = 0;
+        self.liquidatedNotifierRewardAmountChangeInitiated = 0;
+    }
+
+    /// @notice Begins the liquidated notifier reward percentage update process.
+    /// @param _newLiquidatedNotifierRewardPercentage New liquidated notifier
+    ///        reward percentage. This parameter represents the counter of a
+    ///        fraction denominated with 1e18. For example, 3% should be
+    ///        represented as 3*1e16 because 3*1e16/1e18 equals to 0.03
+    function beginLiquidatedNotifierRewardPercentageUpdate(
+        Storage storage self,
+        uint256 _newLiquidatedNotifierRewardPercentage
+    ) internal {
+        /* solhint-disable not-rely-on-time */
+        require(
+            _newLiquidatedNotifierRewardPercentage <=
+                CoveragePoolConstants.FLOATING_POINT_DIVISOR,
+            "Maximum percentage value is 100%"
+        );
+        self
+            .newLiquidatedNotifierRewardPercentage = _newLiquidatedNotifierRewardPercentage;
+        self.liquidatedNotifierRewardPercentageChangeInitiated = block
+            .timestamp;
+        /* solhint-enable not-rely-on-time */
+    }
+
+    /// @notice Finalizes the liquidated notifier reward percentage update process.
+    function finalizeLiquidatedNotifierRewardPercentageUpdate(
+        Storage storage self
+    ) internal {
+        self.liquidatedNotifierRewardPercentage = self
+            .newLiquidatedNotifierRewardPercentage;
+        self.newLiquidatedNotifierRewardPercentage = 0;
+        self.liquidatedNotifierRewardPercentageChangeInitiated = 0;
+    }
```

Comment:
I am thinking we could combine liquidating and liquidated rewards update into two functions given that it is always either amount or percentage. Small detail though, I am good with having it as-is but figured I'll leave the note just in case it helps with some change in the future.